olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -127,10 +127,10 @@ spec:
     - A producer can generate events before a consumer is listening, and a consumer can express an interest in an event or class of events that are not yet being produced.
     - Services can be connected to create new applications without modifying producer or consumer, and with the ability to select a specific subset of events from a particular producer.
 
-    ## Serverless functions
-    Serverless functions extend Knative allowing developers to write functions that let them focus on business logic.
+    ## Knative Functions
+    Knative Functions allow developers to write functions that let them focus on business logic.
```

Comment:
allows? :shrug:

Reply:
Depends if you see it as a component yes, just copied the text from docs jira.

Reply:
Functions, plural, is "allow". Function singular is "allows". I think when we are talking of the product, we are speaking singular (there is only one Knative Functions product). So, this is a long way of saying, "yes, it should be 'allows'". :)